tests/ProjectConfigTest.php

```diff
@@ -65,10 +67,10 @@ protected function setUp()
 
     public function testInit()
     {
-        // Check version
+        // Check supported version
```

Comment:
No need to edit this test case. 
This has nothing to do with support.
All it checks is that project config gets initialized with the same version as in datafile.